src/device-info/device.service.ts

```diff
@@ -0,0 +1,106 @@
+import { REQUEST } from '@nguniversal/express-engine/tokens';
```

Comment:
@joejordanbrown  

This will cause an error if the the application doesn't use server side rendering. I get the below error after importing the 2.0 library to my work project:
```
ERROR in /Users/fanjin/code/medeo-skin/node_modules/angular2-drag-scroll/src/device-
info/device.service.ts (1,25): Cannot find module '@nguniversal/express-engine/tokens'.
```

any idea how to get around it?